tests/src/test/scala/whisk/core/controller/test/RulesApiTests.scala

```diff
@@ -898,7 +898,7 @@ class RulesApiTests extends ControllerTestCommon with WhiskRulesApi {
         val aentity = BadEntity(namespace, aname())
         val rule = WhiskRule(namespace, aname(), afullname(namespace, aname().name), afullname(namespace, aname().name))
         val trigger = WhiskTrigger(namespace, rule.trigger.name)
-        val action = WhiskAction(namespace, rule.action.name, js("??"))
+        val action = WhiskAction(namespace, rule.action.name, js6("??"))
```

Comment:
How about defining a `testKind` in `ControllerTestCommon` which is mapped to `js6` for now, so we can update faster in the future?